flocker/common/script.py

```diff
@@ -53,12 +73,19 @@ def opt_version(self):
         raise SystemExit(0)
     cls.opt_version = opt_version
 
+    # verbose and log-dir should be in a separate decorator
+    # that is only applied to CLI scripts
     def opt_verbose(self):
         """Turn on verbose logging."""
         self['verbosity'] += 1
     cls.opt_verbose = opt_verbose
     cls.opt_v = opt_verbose
 
+    cls.optParameters = getattr(cls, 'optParameters', [])
+    cls.optParameters.append(
+        ['log-dir', None, user_log_dir("Flocker", "ClusterHQ"),
```

Comment:
I'm not sure what a good default log location is so I'd like someone else to look at this.